apps/ecosystem/src/ecosystem.repository.ts

```diff
@@ -1369,4 +1369,134 @@ export class EcosystemRepository {
       throw error;
     }
   }
-}
+
+  async getEcosystemsByOrgId(orgId: string): Promise<IEcosystemData[]> {
```

Comment:
You can use `getAllEcosystemDetails` function instead of creating new method, Please check this function will fits with your flow

Reply:
In the getAllEcosystemDetails function, we are passing pagination and search parameters. However, for the getEcosystems function, we only need the ecosystem data, not the pagination and search parameters. For that purpose I have created a separate function for it.